dockerfiles/test-system/Dockerfile

```diff
@@ -52,7 +54,7 @@ RUN rm -rf ./server/go
 
 RUN --mount=from=uv-image,source=/uv,target=/bin/uv \
     --mount=type=cache,target=/root/.cache/uv \
-    uv pip install .[complete]
+    uv pip install .[complete,postgres]
```

Comment:
install from `dev-requirements.txt`

Reply:
perhaps we can name it "dev-postges" to not be confused with other mlrun features